content/docs/tutorials/windowsbuild.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "FBuilding Qri on Windows"
+description: ""
+date: 2019-06-11T00:00:00-04:00
+section: tutorials
+---
+
+# Building Qri on Windows
+
+*This page was adapted from [a blog post](https://medium.com/@jbutler18/how-to-install-qri-for-windows-83b019c79320) written by friend of the project Jonathan Butler.*
+
+
+**Step 1: Enable Developer Mode** 
+
+<div class="clear"></div>
+
+<div class="diagram large">
+  <img src="/graphics/windows_gifs/Step 0.gif" />
+</div>
+
+If developer mode isn’t enabled, you may experience permission errors during building. To prevent this, enable developer mode by typing “Developer mode” into the search bar and enabling it.
+
+---
+
+**Step 2: Download Go** 
+
+<div class="clear"></div>
+
+<div class="diagram large">
+  <img src="/graphics/windows_gifs/Step 1.gif" />
+</div>
+
+Qri requires Go the programming language and its packages to run. You can download Go from [their official website](https://golang.org/dl/).
+
+---
+
+**Step 3: Download MSYS2**
+
+<div class="clear"></div>
+
+<div class="diagram large">
+  <img src="/graphics/windows_gifs/Step 2.gif" />
+</div>
+
+You can also use your own shell program. However, the creator of the docs tested and used MSYS2. I have also downloaded and used MSYS2 in this example. You can find the MSYS2 download link [here](https://www.msys2.org/).
+
+---
+
+**Step 4: Add the Environment Variables**
+
+<div class="clear"></div>
+
+<div class="diagram large">
+  <img src="/graphics/windows_gifs/Step 3.gif" />
+</div>
+
+In this example, I type “Environment Variable” into the search bar. I add the three paths listed below to my systems environment variables.
+
+```
+PATH=C:go\bin
```

Comment:
These two paths are incorrect, and should be fixed on the original blogpost, otherwise they will cause problems for anyone following these directions. `PATH` is used by the shell to run commands, `GOPATH` is where a developer stores go language source files, so they absolutely should not use the same values.

Here's some more information on `GOPATH`: https://github.com/golang/go/wiki/SettingGOPATH#windows which is official and uses "c:\go-work". Another guide: https://flaviocopes.com/go-gopath/.

Concerning `PATH`, it keeps a list of directories that are searched whenever a user types a command into the shell. For example, if `PATH` has a value like "/usr/bin:/usr/local/bin:/c/Program Files/utils", when a user types `make`, the shell will look for "/usr/bin/make", then "/usr/local/bin/make", then "/c/Program Files/utils/make" and so forth until it finds an executable file to run for that command. By setting `PATH` to "c:go\bin", you're erasing all other paths, meaning most commands will stop working. The correct thing to do is to append to `PATH` so that it's previous list of directories remains. One way to do this is to use "PATH=$PATH:/c/go/bin", the other is to use the GUI as described in the [main Windows build instructions](https://github.com/qri-io/qri#environment-variables). The way this is written sounds like the wrong approach.

For `GOPATH`, the directory where a developer's go source files are stored, typically is something within the user's home directory, like "C:/Users/[username]/go", which would be "/c/Users/[username]/go" in msys2 (replace [username] with the Window user's actual username). Not following this, and using "C:/go" instead would make it very easy to accidentally break the go compiler and toolchain installation, in addition to causing a host of other problems.

Finally, `PATH` also needs to include "$GOPATH/bin" since that's where the `go` compiler will store the executables it builds.

(one final wrinkle, `PATH` on Windows uses ";" (a semi-colon) to separate list entries, since Windows uses ":" (colon) to separate drive letters. msys2 translates this over to be more like POSIX / Linux / OSX systems that use ":" (colon) to separate list entries)

Reply:
Clarified the language here. It's tough, because this issue is legitimately subtle and tricky.